src/components/BooksList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Book from './Book.js';
+import CategoryFilter from './CategoryFilter.js';
+
+const BooksList = ({
+  books, filter, removeBook, changeFilter,
+}) => {
+  const visibleBooks = filter === 'All'
+    ? books
+    : books.filter((book) => book.category === filter);
```

Comment:
Hi, we could optionally do this at the state level so the books coming here are already filtered and keep the component clean too.
[OPTIONAL]